pkg/kubelet/volume/volume_manager.go

```diff
@@ -185,10 +187,17 @@ type volumeManager struct {
 	// desiredStateOfWorldPopulator runs an asynchronous periodic loop to
 	// populate the desiredStateOfWorld using the kubelet PodManager.
 	desiredStateOfWorldPopulator populator.DesiredStateOfWorldPopulator
+
+	// mountedVolulmeToClean is the list of volumes already mounted before
+	// kubelet starts. They need to be cleaned up before reconciler runs
+	mountedVolumeToClean []operationexecutor.MountedVolume
 }
 
 func (vm *volumeManager) Run(stopCh <-chan struct{}) {
 	defer runtime.HandleCrash()
+	glog.Infof("Clean up already mounted volumes before Kubelet starts")
```

Comment:
The mounted volumes can belong to a running pod. Are you trying to unmount all of them...? You need to identify orphaned volumes and only clean up those. Also, if not all pod sources are ready, cleanup cannot be performed.